src/platform/forms/components/review/SavableErrorMessage.jsx

```diff
@@ -0,0 +1,119 @@
+// libs
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+// platform - forms - actions
+import {
+  saveErrors,
+  saveAndRedirectToReturnUrl as saveAndRedirectToReturnUrlAction,
+} from 'platform/forms/save-in-progress/actions';
+import { toggleLoginModal as toggleLoginModalAction } from 'platform/site-wide/user-nav/actions';
+
+import { APP_TYPE_DEFAULT } from 'platform/forms-system/src/js/constants';
+
+// platform - forms components
+import ErrorMessage from 'platform/forms/components/common/alerts/ErrorMessage';
+
+// platform - forms containers
+import SaveFormLink from 'platform/forms/save-in-progress/SaveFormLink';
+
+// platform
+import CallHRC from 'platform/static-data/CallHRC';
+
+// platform - forms - selectors
+import {
+  formSelector,
+  showLoginModalSelector,
+  userSelector,
+} from 'platform/forms/selectors/review';
+import { defaultProps } from 'recompose';
+
+function SavableErrorMessage(props) {
```

Comment:
What do you think about the name `FormSaveErrorMessage`? The error message isn't savable itself; it's reporting an error on the form save.

Reply:
Sounds good, name updated.